concordium-sdk/src/main/java/com/concordium/sdk/transactions/AccountAddress.java

```diff
@@ -32,6 +38,25 @@ public static AccountAddress from(String address) {
         return AccountAddress.from(addressBytes);
     }
 
+    public AccountAddress newAlias(int alias) {
+        if (alias < 0) {
+            throw new NumberFormatException("Alias must be non negative");
+        }
+        if (alias > ALIAS_MAX_VALUE) {
```

Comment:
If you set `ALIAS_MAX_VALUE = 1 << 24` then use `>= here`.

Reply:
I've set the `ALIAS_MAX_VALUE` to `(1 << 24 ) - 1`